docs/sources/configure-client/trace-span-profiles/ruby-span-profiles.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Span profiles with Traces to profiles for Ruby
+menuTitle: Span profiles with Traces to profiles (Ruby)
+description: Learn about and configure Span profiles with Traces to profiles in Grafana for the Ruby language.
+weight: 100
+---
+
+# Span profiles with Traces to profiles for Ruby
+
+![span-profiles screenshot](https://grafana.com/static/img/docs/tempo/profiles/tempo-profiles-Span-link-profile-data-source.png)
+
+## Before you begin
+
+Your applications must be instrumented for profiling and tracing before you can use span profiles. 
+
+* Profiling: Your application must be instrumented with Pyroscopes Ruby SDK. If you haven't done this yet, please refer to the [Ruby (push mode)]({{< relref "../language-sdks/ruby" >}}) guide.
+* Tracing: Your application must be instrumented with OpenTelemetry traces. If you haven't done this yet, please refer to the [OpenTelemetry](https://opentelemetry.io/docs/ruby/getting-started/) guide.
+
+## OpenTelemetry support
+
+Pyroscope can integrate with distributed tracing systems supporting [**OpenTelemetry**](https://opentelemetry.io/docs/instrumentation/ruby/getting-started/) standard which allows you to
+link traces with the profiling data, and find resource usage for specific lines of code for your trace spans
+
+:::note
+ * Only CPU profiling is supported at the moment.
+ * Because of how sampling profilers work, spans shorter than the sample interval may not be captured. Ruby CPU profiler probes stack traces 100 times per second, meaning that spans shorter than 10ms may not be captured.
+:::
```

Comment:
```suggestion
{{< admonition type="note" >}}
 * Only CPU profiling is supported at the moment.
 * Because of how sampling profilers work, spans shorter than the sample interval may not be captured. Ruby CPU profiler probes stack traces 100 times per second, meaning that spans shorter than 10ms may not be captured.
{{< /admonition >}}
```